tests/dummy/app/controllers/application.js

```diff
@@ -1,9 +1,12 @@
 import Ember from 'ember';
 
-export default Ember.Controller.extend({
+const { Controller, A, computed } = Ember;
```

Comment:
This is going to be outdated syntax extremely soon now that addons are starting to update to the new module api.